client/galaxy/scripts/components/UtcDate.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+    <span class="utc-time" v-if="mode == 'date'" :title="elapsedTime">
+        {{ date }}
+    </span>
+    <span v-else :title="date">
+        {{ elapsedTime }}
+    </span>
+</template>
+
+<script>
+export default {
+    props: {
+        date: {
+            type: String,
+            required: true
+        },
+        mode: {
+            type: String,
+            default: "date" // or elapsed
+        }
+    },
+    computed: {
+        elapsedTime: function() {
```

Comment:
We have moment as dependency right now (due to handsontable, so if we're changing that out this is moot), and if you wanted to use that it'd simplify the bits below quite a bit.

Reply:
:+1: for momentjs, pure JS is horrendous with dates